kie-wb/kie-wb-webapp/src/main/java/org/kie/workbench/backend/LogoutHelper.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+package org.kie.workbench.backend;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LogoutHelper {
+
+    public static final Logger logger = LoggerFactory.getLogger(LogoutHelper.class);
+
+    public static final void logout(HttpServletRequest request, HttpServletResponse response) throws ServletException {
+        try {
+            request.logout();
+            HttpSession session = request.getSession(false);
+            if (session != null) {
+                session.invalidate();
+            }
+            // JBPM-5378: In the latest EAP 7.x versions SSO does not works if the cookie is not removed
+            Cookie cookie = new Cookie("JSESSIONID", null);
+            cookie.setPath("/dashbuilder");
```

Comment:
My only comment is; what is the guarantee that the web-context for Dash Builder is *always* ```/dashbuilder```? Would it be preferential to allow this to be overridden with a System Property? Otherwise I'm happy.